rclcpp/src/rclcpp/time_source.cpp

```diff
@@ -246,8 +246,8 @@ void TimeSource::on_parameter_event(const rcl_interfaces::msg::ParameterEvent::S
   }
   // Filter for only 'use_sim_time' being added or changed.
   rclcpp::ParameterEventsFilter filter(event, {"use_sim_time"},
-    {rclcpp::ParameterEventsFilter::EventType::NEW,
-      rclcpp::ParameterEventsFilter::EventType::CHANGED});
+  {rclcpp::ParameterEventsFilter::EventType::NEW,
+    rclcpp::ParameterEventsFilter::EventType::CHANGED});
```

Comment:
@rotu uh, would leaving the opening curly brace on line 248 improve things a bit?

Reply:
Maybe, but there's a bigger issue here - for some reason, Uncrustify is not recognizing this as a mult-line function call. There should be a line break after `filter`:

https://github.com/ros2/rclcpp/blob/ee6ab95cfc8ecab353d4cf1efdb81de237424f11/rclcpp/src/rclcpp/time_source.cpp#L248-L250

Reply:
submitted https://github.com/ament/ament_lint/issues/231